Docker information added

MadRussian opened the pull request:
- Added how to run lighterpack in a docker container.
- Also added a docker directory with the Dockerfile that was used to
  generate the image which is in docker hub.

Added the image that is necessary for this in Docker Hub:
https://hub.docker.com/r/madrussian/lighterpack/

xeor commented:
Hi

Is the docker-image at madrussian/ligherpack suposed to work out of the box? It doesn't start `mongod`, and in run privileged (which is odd...?)..

I'm looking for a docker image for this, but before I make another one, I want to make sure that I'm not duplicating someone else work..

I want to make it using alpine, or the official nodejs image..

Comments?

xeor commented:
Alternative, I can make a version that you can build yourself, instead of anyone hosting it..

MadRussian commented:
You are correct there is an issue with the Dockerfile as you mention above which will need to be fixed. Here's the diff to the Dockerfile which should fix it although will need some additional testing for it to make sure there aren't any other issues. As a short term here's a patch to the Dockerfile:
```
diff --git a/docker/Dockerfile b/docker/Dockerfile
index 00af796..317b1a6 100644
--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -20,4 +20,4 @@ RUN cd /var/www/lighterpack && npm install
 EXPOSE 3000
 
 WORKDIR /var/www/lighterpack
-CMD ["nodejs", "app.js"]
+CMD service mongodb start && nodejs app.js
```

I'll push this up a bit later once I do some additional testing for this, but this should allow it to work for anyone that needs it sooner. If you run into other issues either post it here or create a separate issue. As far as hosting anyone can do that I just provided a starter in case anyone needed it.

As far as privileged that is not necessary and should work w/o that. I've updated dockerhub to say that. Also updated dockerhub with an updated image which has the changes to the Dockerfile mentioned above.
